src/@next/components/atoms/DropdownMenu/types.ts

```diff
@@ -0,0 +1,10 @@
+export interface MenuItem {
+  content: React.ReactNode;
+  onClick: () => void;
+}
+
+export interface IProps {
+  type: "hoverable" | "clickable";
+  header: React.ReactNode;
+  content: MenuItem[];
```

Comment:
I would try to find a better name,  content is used in MenuItem too.

Reply:
I think `items` can fit better here.

Reply:
Changed to `items`